config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-main.yaml

```diff
@@ -206,3 +206,37 @@ periodics:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
     testgrid-tab-name: capz-periodic-e2e-full-main
     testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
+- name: periodic-cluster-api-provider-azure-e2e-aks-main
+  decorate: true
+  decoration_config:
+    timeout: 4h
+  interval: 12h
```

Comment:
should we make this hourly just for today so we can gather data? and then reduce frequency after 24 hours

Reply:
good idea, done